src/components/EditActivityPage/EditActivityPage.test.js

```diff
@@ -42,6 +42,6 @@ describe('EditActivityPage', () => {
   it('should handle onRemoveActivities', () => {
     wrapper.find('button').simulate('click');
     expect(historySpy.push).toHaveBeenLastCalledWith(ACTIVITY_PLAN_ROUTE);
-    expect(onRemoveActivitySpy).toHaveBeenLastCalledWith(testState.activities.items[1].id);
+    expect(onStartRemoveActivitySpy).toHaveBeenLastCalledWith(testState.activities.items[1].id);
```

Comment:
why is there usage of `toHaveBeenLastCalledWith` instead of simply `toHaveBeenCalled` ? Are you expecting more invokation of this method ? There should be only one